src/main/java/Main.java

```diff
@@ -5,7 +5,7 @@
 	public static void main(String[] args) throws InterruptedException {
 		CoffeeMaker markIV = new CoffeeMaker();
 
-		markIV.startProcess();
+		markIV.startProcess(4);
 
```

Comment:
The quantity of cups should be selectioned from comboBox displayed in the frame.